src/fn/fn-popover.scss

```diff
@@ -0,0 +1,127 @@
+@import "./fn-settings";
+
+$block: #{$fn-namespace}-popover;
+
+$fn-popover-arrow-size: 0.75rem;
+$fn-popover-arrow-offset: -0.25rem;
+$fn-popover-arrow-margin: 1.5rem;
+$fn-popover-arrow-middle-calc: calc(50% - 0.25rem);
+
+.#{$block} {
+  @include fn-reset();
+  @include fn-flex(column);
+
+  background-color: $fn-color-white;
+  border-radius: 0.75rem;
+  padding: 1rem 1.5rem;
+  position: relative;
+  filter: drop-shadow(0 0 0.125rem rgba(0, 0, 0, 0.25)) drop-shadow(0 0.5rem 1rem rgba(0, 0, 0, 0.25));
+
+  &__title {
+    @include fn-reset();
+
+    font-size: 1.125rem;
+    font-weight: bold;
+    line-height: 1.375rem;
+  }
```

Comment:
In my opinion `.fn-popover__title` should be a container as the user might want at some point to add icon, status indicator, etc. not just text. Then inside it you should have the text, which by default goes on multi line if too long and have a modifier class for truncate. Something like:
```
<div class="fn-popover__title>
    <div class="fn-popover__text>Title</div>
</div>
```

And then the css would look something like:
```
.fn-popover__title {
    @include fn-reset();

    padding: 0.75rem;
    width: 100%;
}

.fn-popover__title {
    @include fn-reset();

    font-size: 1.125rem;
    font-weight: bold;
    line-height: 1.375rem;

    &--truncate {
         @include fn-ellipsis()
    }
}
```

Reply:
How about:

```
  &__title {
    @include fn-reset();

    padding: $fn-popover-padding;
    width: 100%;

    .#{$block}__text {
      @include fn-reset();

      font-size: 1.125rem;
      font-weight: bold;
      line-height: 1.375rem;

      &--truncate {
        @include fn-ellipsis();
      }
    }
  }
```
with __text specifically needing to be a child of __title?